flask_securest/rest_security.py

```diff
@@ -228,9 +215,25 @@ def authenticate(authentication_providers, auth_info):
     if not user:
         raise Exception('Unauthorized')
 
+    set_request_user(user)
     return user
 
 
+def get_request_user():
+    request_user = None
+    # TODO is there a nicer way to do this?
+    request_ctx = _request_ctx_stack.top
+    if hasattr(request_ctx, 'user'):
```

Comment:
could use getattr(request_ctx, 'user') instead, and throw exception if user is not there